### ISSUE TITLE
NVDA screen reader stops reading list of classes in file explorer

### ISSUE THREAD

**Issue by jeanbrazcosta** [microsoft/vscode#168463]
Type: <b>Bug</b>

After the last update, when browsing the classes in the file explorer, the NVDA screen reader stops reading the class names and only works again if you switch windows and go back to VsCode.

This behavior started to occur since I updated to the latest version of VS Code.

I can't say if the same problem occurs in the insider version, because I haven't been able to test it yet.

Another way I was able to reproduce it was in the List Audio Cues, when I was up and down with the arrows to hear the new sounds.

I emphasize that this failure is intermittent.

VS Code version: Code 1.74.0 (5235c6bb189b60b01b1f49062f4ffa42384f8c91, 2022-12-05T16:38:16.075Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: No
Remote OS version: Linux x64 5.15.74.2-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8365U CPU @ 1.60GHz (8 x 1896)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.72GB (5.92GB free)|
|Process Argv|--crash-reporter-id b7a1990b-0b40-4fbd-b9b4-948e24d1252c|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: ubuntu|
|OS|Linux x64 5.15.74.2-microsoft-standard-WSL2|
|CPUs|Intel(R) Core(TM) i5-8365U CPU @ 1.60GHz (4 x 1896)|
|Memory (System)|11.69GB (8.96GB free)|
|VM|0%|
</details><details><summary>Extensions (17)</summary>

Extension|Author (truncated)|Version
---|---|---
remote-wsl|ms-|0.72.0
cucumber-official|Cuc|1.6.0
xml|Dot|2.5.1
gitlens|eam|13.1.1
mongodb-vscode|mon|0.9.5
vscode-thunder-client|ran|2.1.2
fabric8-analytics|red|0.3.6
java|red|1.13.0
sonar|son|0.0.3
sonarlint-vscode|Son|3.12.0
intellicode-api-usage-examples|Vis|0.2.6
vscodeintellicode|Vis|1.2.29
vscode-java-debug|vsc|0.47.0
vscode-java-dependency|vsc|0.21.1
vscode-java-pack|vsc|0.25.7
vscode-java-test|vsc|0.37.1
vscode-maven|vsc|0.40.1

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
pyindex848:30577860
nodejswelcome1:30587005
282f8724:30602487
gswce1:30612156
3d0df643:30613357
dbltrim-noruby:30604474
f6dab269:30613381
fim-prod:30623723
ejf25101:30620729

```

</details>

**Comment by jvesouza** [comment]
I tried to reproduce the problem using orca in my linux environment but the problem did not appear.
I tested using the Insiders version and the 1.74 version and the result was the same.

@jeanbrazcosta Would it be possible to describe the steps taken to reproduce the problem?

**Comment by jeanbrazcosta** [comment]
@jvesouza step to reproduce:

1: Press control + shift + f, in file explorer, use the arrows and browse your classes, here after going through some files, the reader stopped reading.

I don't have linux to test, here it only happened with NVDA.

The rpojects I'm working on are quite large, but I've always opened them in previous versions of VS Code and they didn't have this behavior.

**Comment by jeanbrazcosta** [comment]
@meganrogge I tested it today, when the update appeared for me

**Comment by jvesouza** [comment]
@jeanbrazcosta  I've some questions:

Are you working with a java project?
Approximately how many classes do you have in the project?
In the instructions to reproduce the problem you instruct me to press ctrl+shift+f, but I think the correct thing would be ctrl+shift+e, right?

Maybe I couldn't reproduce the problem because I used a project with few classes.

**Comment by jeanbrazcosta** [comment]
@jvesouza You were correct I'm sorry I entered the wrong command it was really control + shift + e.

About this problem I found the solution, I uninstalled my version and installed it again, when I managed to test it in the Insiders version and I noticed that this problem does not occur.

Therefore, I will close this issue and if it happens again, I don't know if it is possible but I will try to generate a log and send it along with the request
